InternetHospital.WebApi/Controllers/PatientController.cs

```diff
@@ -28,6 +28,36 @@ public PatientController(UserManager<User> userManager, IPatientService patientS
             _patientService = patientService;
         }
 
+        [HttpGet("GetProfile")]
+        public async Task<IActionResult> GetPatient()
+        {
+            var patientId = User.Identity?.Name;
+            if (patientId != null)
+            {
+                var patient = await _userManager.FindByIdAsync(patientId);
+
+                var returnPatient = await _patientService.Get(int.Parse(patientId));
```

Comment:
Unsafe int.Parse() better not to check if patientId is null, but to check if it can be parsed into int.